File: bee1101.c
#include <stdio.h>
 
int main() {
 
    int m, n, soma;
    while(1 == 1){
        soma = 0;
        scanf("%d%d", &m,&n);
        if( m <= 0 || n <= 0){
            break;
        }
        if(m < n){
            for(; m <= n; m++){
                printf("%d ", m);
                soma = soma + m;
            }
        }
        else{
            for(; n <= m; n++){
                printf("%d ", n);
                soma = soma + n;
            }
        }
        printf("SUM=%d \n", soma);
    }
 
    return 0;
}

File: bee1101.py
while True:
    soma = 0
    try:
      m,n = map(int, input().split())
    except EOFError:
      print('EOF when reading a line')
      
    if m <= 0 or n <= 0:
      break
    if m < n:
      for i in range(m,n + 1):
        print(i, end=" ")
        soma += i
    else:
      for i in range(n,m + 1):
        print(i, end=" ")
        soma += i     
    print(f' SUM={soma}')
